serenata_toolbox/chamber_of_deputies/dataset.py

```diff
@@ -15,18 +15,30 @@ def __init__(self, path, years=AVAILABLE_YEARS):
 
     def fetch(self):
         base_url = "http://www.camara.leg.br/cotas/Ano-{}.csv.zip"
+        retrieved_files = []
 
         for year in self.years:
             zip_file_path = os.path.join(self.path, "Ano-{}.zip".format(year))
             url = base_url.format(year)
             urlretrieve(url, zip_file_path)
-            zip_file = ZipFile(zip_file_path, 'r')
-            zip_file.extractall(self.path)
-            zip_file.close()
+            retrieved_files.append(zip_file_path)
+
+            self._extract_zip_file(zip_file_path)
             os.remove(zip_file_path)
 
+        DATASETS_FORMAT = 'datasets-format.html'
```

Comment:
What about moving the file name and the URL as class constants? ([example](https://github.com/datasciencebr/serenata-toolbox/blob/master/serenata_toolbox/datasets/remote.py#L11-L13))

Reply:
It is used just inside fetch method. I can't see the benefits of it.

The idea is have a project pattern to have constants at class level?